app/web/features/search/FilterDialog.tsx

```diff
@@ -86,8 +86,8 @@ interface FilterDialogProps {
   isOpen: boolean;
   onClose(): void;
   queryName: undefined | string;
-  setQueryName: Dispatch<SetStateAction<undefined | string>>;
-  setLocationResult: any;
+  setQueryName: Dispatch<SetStateAction<string>>;
+  setLocationResult: Dispatch<SetStateAction<GeocodeResult>>;
```

Comment:
thanks, I always use `any` when I try to set the type multiple times and fail (and after I forget to change it 😅)

Reply:
Haha I gotcha, TS can be a pain sometimes.

I think some of the linting PRs I just opened will hopefully make it easier not to forget. I was also losing track of mine and only saw after Gitlab ran.